tests/c1-integration/test/document.test.ts

```diff
@@ -40,6 +48,16 @@ const CONTAINER_OPTS: EnvironmentOptions = {
   testPort: 5223,
 }
 
+const FLIGHT_OPTIONS: ClientOptions = {
```

Comment:
it looks like all these tests use the same options. you could define this somewhere shared (index.ts in this case) and import them but it's also kind of nice to keep config close to tests